match/spec/storage/s3_storage_spec.rb

```diff
@@ -110,3 +110,38 @@
     end
   end
 end
+
+describe Match::Storage::S3Storage do
```

Comment:
Can we put this code inside the main `describe Match::Storage::S3Storage do` that is already declared in this file? 🙏

Reply:
thank you!

Thought I wrote a lot of Ruby on Rails code, I'm not familiar with rspec conventions.
Updated these tests with the help of ChatGPT.

Now they are all green.